docs/index.rst

```diff
@@ -0,0 +1,29 @@
+Welcome to plankton's documentation!
+====================================
+
+Plankton is a Python package that makes it easy to develop complex stateful device simulations. It
+is licensed under GPL version 3 and the source is available on github_, where you are welcome to
+open new issues so the package can improve.
+
+If you're new to Plankton reading the :ref:`user_guide` is a good start. Once you plan on writing
+your own device it's time to go through the :ref:`developer_guide`. The :ref:`api_documentation`
```

Comment:
On the one hand it would also save the labels, but on the other hands probably labels are not the worst thing here in case we change the structure of this. I'm leaning a bit more towards the `:doc:`-solution you suggested.